src/maingoblint.ml

```diff
@@ -621,7 +621,9 @@ let do_gobview cilfile =
       (* marshal timing statistics *)
       let stats = Fpath.(run_dir / "stats.marshalled") in
       Serialize.marshal (Timing.Default.root, Gc.quick_stat ()) stats;
-  )
+    )
+    else
+      Logs.error "Warning: Cannot locate GobView."
```

Comment:
It's a bit confusing to have an error say it's a warning:
```suggestion
      Logs.error "Cannot locate GobView."
```
I might have caused this on master, but this is a good opportunity to fix it.